dev/com.ibm.ws.kernel.boot_fat/fat/src/com/ibm/ws/kernel/service/ServerEndpointControlMBeanTest.java

```diff
@@ -0,0 +1,225 @@
+
+        } catch (MBeanException e) {
+            Assert.assertEquals(PAUSEABLE_EXCEPTION_CLASS, e.getCause().getClass().getName());
+        }
+        assertNotNull(server.waitForStringInLog("CWWKE0933W", logWaitTimeout));
+
+        Log.exiting(c, METHOD_NAME);
+    }
+
+    /**
+     * Tests that the expected exception is thrown if the MBean resume method is issued when no pauseable components exist
+     *
+     * @throws Exception
+     */
+    @ExpectedFFDC(PAUSEABLE_EXCEPTION_CLASS)
+    @Test
+    public void testResumeNoPauseableComponents() throws Exception {
+        final String METHOD_NAME = "testResumeNoPauseableComponents";
+        Log.entering(c, METHOD_NAME);
+
+        try {
+            mbean.resume("");
+            fail("Didn't get expected exception from attempting to resume a server with no pauseable components");
+
+        } catch (MBeanException e) {
+            Assert.assertEquals(PAUSEABLE_EXCEPTION_CLASS, e.getCause().getClass().getName());
+        }
+        assertNotNull(server.waitForStringInLog("CWWKE0934W", logWaitTimeout));
+
+        Log.exiting(c, METHOD_NAME);
+    }
+
+    /**
+     * Tests that the expected exception is thrown if the MBean pause method is issued with an invalid target.
+     *
+     * @throws Exception
+     */
+    @ExpectedFFDC(PAUSEABLE_EXCEPTION_CLASS)
+    @Test
+    public void testPauseInvalidTarget() throws Exception {
+        final String METHOD_NAME = "testPauseInvalidTarget";
+        Log.entering(c, METHOD_NAME);
+
+        try {
+            mbean.pause("inValidTargetName!");
+            fail("Didn't get expected exception from attempting to pause a server with an invalid target");
+
+        } catch (MBeanException e) {
+            Assert.assertEquals(PAUSEABLE_EXCEPTION_CLASS, e.getCause().getClass().getName());
+        }
+        assertNotNull(server.waitForStringInLog("CWWKE0935W", logWaitTimeout));
+
+        Log.exiting(c, METHOD_NAME);
+    }
+
+    /**
+     * Tests that the expected exception is thrown if the MBean resume method is issued with an invalid target.
+     *
+     * @throws Exception
+     */
+    @ExpectedFFDC(PAUSEABLE_EXCEPTION_CLASS)
+    @Test
+    public void testResumeInvalidTarget() throws Exception {
+        final String METHOD_NAME = "testResumeInvalidTarget";
+        Log.entering(c, METHOD_NAME);
+
+        try {
+            mbean.resume("inValidTargetName!");
+            fail("Didn't get expected exception from attempting to resume a server with an invalid target");
+
+        } catch (MBeanException e) {
+            Assert.assertEquals(PAUSEABLE_EXCEPTION_CLASS, e.getCause().getClass().getName());
+        }
+        assertNotNull(server.waitForStringInLog("CWWKE0936W", logWaitTimeout));
+
+        Log.exiting(c, METHOD_NAME);
+    }
+
+    private static MBeanServerConnection getMBeanServer(String serverRoot) throws Exception {
```

Comment:
Can you just use: MBeanServer mbs = ManagementFactory.getPlatformMBeanServer(); instead of this method?

Reply:
Ignore this comment, given that this test is not running on the server